reporting_test.go

```diff
@@ -81,9 +81,7 @@ func TestErrorReporting(t *testing.T) {
 			},
 			expectedErrs: [][]string{
 				{
-					"test.proto:5:41: syntax error: unexpected \"enum\", expecting ';' or '.'",
-					"test.proto:5:69: syntax error: unexpected ';', expecting '='",
-					"test.proto:7:53: syntax error: unexpected '='",
+					"test.proto:5:41: expected ';'",
```

Comment:
Hmm, this is confusing. Why are we reporting fewer things now? The point of this test was to demonstrate reporting multiple errors with a single parse attempt (vs. failing fast).

Reply:
(hmm, no, the D is missing a ';' too)

Reply:
I figured it out, it was panicing because I forgot to update the new function

Reply:
Fixed (and I double checked that all new nil cases have test coverage)